pkg/util/utils.go

```diff
@@ -324,16 +324,21 @@ func Killall(processName string) {
 }
 
 // InterfaceSliceToStringSlice converts a slice of interfaces to a slice of strings
-func InterfaceSliceToStringSlice(v []interface{}) ([]string, error) {
-	raw := make([]string, len(v))
-	for i := range v {
-		if s, ok := v[i].(string); ok {
-			raw[i] = s
-		} else {
-			return nil, fmt.Errorf("%v is not a string", v[i])
+func InterfaceSliceToStringSlice(input interface{}) ([]string, error) {
+	switch v := input.(type) {
+	case []interface{}:
+		raw := make([]string, len(v))
+		for i := range v {
+			if s, ok := v[i].(string); ok {
+				raw[i] = s
+			} else {
+				return nil, fmt.Errorf("%v is not a string", v[i])
+			}
 		}
+		return raw, nil
+	default:
+		return nil, fmt.Errorf("unexpected type: %T", input)
```

Comment:
I added a new type check here because DDEV threw panic on this:

```yaml
hooks:
    post-start:
        - exec:
          exec_raw: ls -la # the command should look like this [ls, la]
```

```
$ ddev start
...
panic: interface conversion: interface {} is string, not []interface {}
...
```

But it should show an error:
```
$ ddev start
...
Invalid exec/exec_raw value, not executing it: unexpected type: string 
Failed to start d11: unable to create task from map[exec:<nil> exec_raw:ls -la] 
```